csrc/pytorch_extension_utils.h

```diff
@@ -14,10 +14,13 @@
  * limitations under the License.
  */
 #pragma once
+#include <ATen/ATen.h>
+#include <ATen/cuda/CUDAContext.h>
```

Comment:
What functions did we rely on `<ATen/ATen.h>` and `<ATen/cuda/CUDAContext.h>`?
The main reason I want to keep the include header as small as possible is to reduce compilation time, so it's better if we only rely on minimal headers.